docs/sensors/lidar_2d/hokuyo_ust20_lx.mdx

```diff
@@ -15,10 +15,9 @@ import Support from "/components/support.mdx";
 
 ## Item Numbers
 
-| Description                      |                         CPR item                         |
-| :------------------------------- | :------------------------------------------------------: |
-| Hokuyo UST20-LX sensor           | [016359](/assets/pdf/clearpath_robotics_016359-TDS1.pdf) |
-| Hokuyo UST20-LX sensor, PACS kit |                                                          |
+| Description            |                         CPR Item                         | Sales Kit | Installation Kit, With Fasteners |
+| :--------------------- | :------------------------------------------------------: | :-------: | :------------------------------: |
+| Hokuyo UST20-LX sensor | [016359](/assets/pdf/clearpath_robotics_016359-TDS1.pdf) |           |                                  |
```

Comment:
Assuming that the kit information is intentionally blank here? Should we add a placeholder like "N/A" or "Coming soon"? Up to you.

Reply:
I have added the kit item numbers for the UST20-LX.